front/src/App.js

```diff
@@ -58,6 +58,12 @@ import { BrowserRouter as Router, Route } from "react-router-dom";
 import UserProfile from "./sideBard/UserProfile";
 
 class App extends Component {
+  checkAuthorizedRouting() {
```

Comment:
i really like this solution, nicely done!

Reply:
Thank you (: